calico/felix/fiptables.py

```diff
@@ -288,8 +306,11 @@ def _finish_msg_batch(self, batch, results):
 
     def _delete_best_effort(self, chains):
         """
-        Try to delete all the chains in the input list.
+        Try to delete all the chains in the input list. Any errors are silently
+        swallowed.
         """
+        # PLW: Do we really need the retry logic? It feels to me that we could
```

Comment:
It allows us to keep making progress before we've completed our first cleanup (in case some chains are keeping something referenced that we're not expecting) or if someone makes a manual change that keeps one of our chains referenced by mistake.

I see it as analogous to using call_silent to delete ipsets except with the optimisation of deleting as many as possible in batches.